readme.md

```diff
@@ -253,6 +253,7 @@ namespace Azure.Service.Models
 ### Change a model namespace
```

Comment:
We should also change the title for this to `Change a model or client namespace`

Reply:
done